core/db/migrate/20170412103617_transform_tax_rate_category_relation.rb

```diff
@@ -0,0 +1,41 @@
+class TransformTaxRateCategoryRelation < ActiveRecord::Migration
+  def up
+    create_table :spree_tax_rates_categories do |t|
+      t.integer :tax_category_id, index: true
+      t.integer :tax_rate_id, index: true
+    end
+
+    add_foreign_key :spree_tax_rates_categories, :spree_tax_categories, column: :tax_category_id
+    add_foreign_key :spree_tax_rates_categories, :spree_tax_rates, column: :tax_rate_id
+
+    Spree::TaxRate.find_each do |tax_rate|
+      Spree::TaxRateCategory.create!(
```

Comment:
Solidus follows the practice of defining all ActiveRecord::Base classes in the migration so that future model changes/renames/etc don't break migrations.

E.g. [this](https://github.com/solidusio/solidus/blob/322a83962e8a8af2a270f82cbc78637e1a528939/core/db/migrate/20160608180751_ensure_store_on_orders.rb)

You'll want to add `class TaxRate < ActiveRecord::Base ...` and `class TaxRateCategory < ActiveRecord::Base ...` (without the `Spree` prefix) to the migration here